datadog_checks_base/datadog_checks/base/checks/openmetrics/base_check.py

```diff
@@ -34,6 +35,14 @@ class OpenMetricsBaseCheck(OpenMetricsScraperMixin, AgentCheck):
 
     DEFAULT_METRIC_LIMIT = 2000
 
+    HTTP_CONFIG_REMAPPER = {
+        'ssl_verify': {'name': 'tls_verify'},
+        'ssl_cert': {'name': 'tls_cert'},
+        'ssl_private_key': {'name': 'tls_private_key'},
+        'ssl_ca_cert': {'name': 'tls_ca_cert'},
+        'prometheus_timeout': {'name': 'timeout'},
```

Comment:
Mean we need to update all integrations config examples?

Reply:
Yes, though they'd still work of course so no rush